benchmarks/viscoelastic_sheared_torsion/viscoelastic_sheared_torsion.prm

```diff
@@ -0,0 +1,169 @@
+# the analytical stress and the aspect stress component need to be multiplied with cos(omega*t). Note that in the
+# Farrington paper the vertical axis is the y-axis, such that their stress component xy is aspect's stress
+# component xz.
+# The fit of the numerical stress to the analytical solution can be enhances by choosing a smaller time step size.
+
+
+# Global parameters
+set Dimension                                  = 3
+set Start time                                 = 0
+set End time                                   = 2
+set Use years in output instead of seconds     = false
+set Resume computation                         = auto
+set CFL number                                 = 1.0
+set Maximum time step                          = 0.01
+set Output directory                           = output
+set Pressure normalization                     = surface
+set Surface pressure                           = 0.
+set Nonlinear solver scheme                    = single Advection, iterated Stokes
+set Max nonlinear iterations                   = 30
+set Nonlinear solver tolerance                 = 1e-5
+set Max nonlinear iterations in pre-refinement = 0
+
+
+# Solver settings
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Number of cheap Stokes solver steps = 200
+    set Linear solver tolerance             = 1e-9
+  end
+end
+
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X extent  = 1.0
+    set Y extent  = 1.0
+    set Z extent  = 1.0
+    set X repetitions = 5
+    set Y repetitions = 5
+    set Z repetitions = 5
+    set Box origin X coordinate = -0.5
+    set Box origin Y coordinate = -0.5
+  end
+end
+
+subsection Mesh refinement
+  set Strategy                           = maximum refinement function
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+  subsection Maximum refinement function
+    set Function expression              = 0
+  end
+end
+
+
+# Velocity boundary conditions
+subsection Boundary velocity model
+  set Prescribed velocity boundary indicators = 0:function, 1:function, 2:function, 3:function, 4:function, 5:function
+  subsection Function
+  # circular rotation plus shearing until t=0.5
+  set Function expression = if(t<0.5, -sin(atan2(y,x))*sqrt(y*y+x*x)*42/360 + 0.3*z, -sin(atan2(y,x))*sqrt(y*y+x*x)*42/360) ;\
+                            cos(atan2(y,x))*sqrt(y*y+x*x)*42/360 ;\
+                            0
+  end
+end
+
+# Formulation classification
+subsection Formulation
+  set Enable elasticity = true
+end
+
+# Gravity model
+subsection Gravity model
+  set Model name  = vertical
+  subsection Vertical
+    set Magnitude = 1
+  end
+end
+
+# Number and name of compositional fields
+subsection Compositional fields
+  set Number of fields = 7
+  set Names of fields  = ve_stress_xx, ve_stress_yy, ve_stress_zz, ve_stress_xy, ve_stress_xz, ve_stress_yz, goo
```

Comment:
Why is there a field goo? Can it be removed?

Reply:
same here, copied it from the simple shear BM, but was wondering there as well, if it could be removed and Background composition is taken instead. Should I clean up the simple shear BM too?